File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.cpp
class Solution {
public:
    int longestConsecutive(vector<int>& nums) {
        if(nums.empty()) return 0;
       map<int,int>mp;
       for(int i=0;i<nums.size();i++) mp[nums[i]]++;
       int cnt=0,ans=1;
       auto tmp=mp.begin();
       int prev=tmp->first;
       for(auto it:mp){
        if(prev==it.first or prev+1==it.first){
            cnt++;
            ans=max(ans,cnt);
            prev=it.first;
        }
        else{
            cnt=1;
            prev=it.first;

        }
       }
       return ans;
    }
  
};

File: Data Structures & Algorithms/counting-bits/submission-0.py
class Solution:
    def countBits(self, n: int) -> List[int]:
        vec=[]
        for i in range(0,n+1):
            cnt=0
            for j in range(0,11):
                if(i & 1<<j): cnt+=1
            vec.append(cnt)
        return vec
        